.github/workflows/release-please.yaml

```
@@ -19,7 +19,7 @@ jobs:
         if: ${{ steps.release.outputs.release_created }}
       - uses: actions/setup-node@v1
         with:
-          node-version: 12
+          node-version: 16
```

Let's bump to the lst, #177 this PR actually require 18 minimal

```suggestion
          node-version: 20
```